Deploy to the test server (#530)
decentraland/documentation: _posts/development-guide/2018-01-07-publishing.md

2.  The `dcl deploy` command links these assets to the LAND parcel specified in your **scene.json** file. Whenever you redeploy your scene, the CLI will update your LAND smart contract, if needed, to point to the most recent content available on the content servers.

The information on each copy of the server is verifiable, as each scene is signed by the LAND owner's hash. This means that someone hosting a copy of the server won't be able to tamper with the content to display something illegitimate. The community can also vote to approve or remove any of these servers using the DAO.

## The test server

> You can deploy content to the test catalyst server to run full tests with multiple users, the sourrounding scenes, and an environment that is identical to production. The test server is identical to all other catalyst servers, the difference is that the content that is deployed to this server isn't propagated to the others. Content deployed to other servers on the other hand does get propagated to this server, so surrounding scenes should look as they will in production.

Reviewer: ```suggestion
You can deploy content to the test catalyst server to run full tests with multiple users, the surrounding scenes, and an environment that is identical to production. The test server is identical to all other catalyst servers, the difference is that the content that is deployed to this server isn't propagated to the others. Content deployed to other servers on the other hand does get propagated to this server, so surrounding scenes should look as they will in production.
```

To deploy to the test server, run:

`dcl deploy --target peer-testing.decentraland.org`

> > Note: The same permissions apply as in production. You must be owner or have permissions on the parcels that you're deployng to.

Reviewer: ```suggestion
> Note: The same permissions apply as in production. You must be the owner or have permission on the parcels that you're deploying to.
```

Players are never directed to this server, the only way to access it is to explicitly provide a URL parameter to connect to it. 

To enter the content server, add `&CATALYST=peer-testing.decentraland.org` to the Decentraland URL

play.decentraland.org/&CATALYST=peer-testing.decentraland.org

> If you're working in a confidential project that you don't want to unveil until launch, note that the test server is relatively hidden from players, but anyone explicitly using the test server's URL could potentially run into it.

Reviewer: ```suggestion
If you're working on a confidential project that you don't want to unveil until launch, note that the test server is relatively hidden from players, but anyone explicitly using the test server's URL could potentially run into it.
```
